README.md

```diff
@@ -35,7 +35,8 @@ Ein Buchungsvorschlag wird as template genommen um eine Buchung zu erstellen.
 ![Klassendiagramm (Erweitert)](./sw-eng-04.drawio.svg)
 
 #### Erklärung
-<!-- TODO -->
+
+Ein Benutzer könnte die Kreditkarte einer anderen Person nutzen, deshalb gibt es keine Aggregation zwischen der Karte und dem Nutzer. Jede Buchung basiert auf einem Buchungsvorschlag, daher ist der Vorschlag ein wichtiger Teil der Buchung. Die Rückgabe-/Abgabedaten und -orte sind Teil des Buchungsvorschlags, da sie als Parameter für die Vermittlungssuche verwendet werden. Das "Reduzierung" Attribut in den Buchungen enthält Informationen wie Aktionen oder Reduzierungen der Selbstbeteiligung gegen eine zusätzliche Gebühr. Es besteht eine viele-zu-viele Assoziation zwischen Buchungsvorschlägen und E-Mails, da ein Kunde mehrere E-Mails für einen Vorschlag senden und erhalten kann.
```

Comment:
```suggestion
Ein Benutzer könnte die Kreditkarte einer anderen Person nutzen, daher gibt es keine Aggregation zwischen der Karte und dem Nutzer. Jede Buchung basiert auf einem Buchungsvorschlag, daher ist der Vorschlag ein wichtiger Teil der Buchung. Die Rückgabe-/Abgabedaten und -orte sind Teil des Buchungsvorschlags, da sie als Parameter für die Vermittlungssuche verwendet werden. Das Attribut "Reduzierung" in den Buchungen enthält Informationen wie Aktionen oder Reduzierungen der Selbstbeteiligung gegen eine zusätzliche Gebühr. Es besteht eine viele-zu-viele-Assoziation zwischen Buchungsvorschlägen und E-Mails, da ein Kunde mehrere E-Mails zu einem Vorschlag senden und erhalten kann.
```

Reply:
Just to be clear, the E-Mails are sent from the user to enquire about (or confirm) shown offers. Could you please elaborate on the use of _zu einem_, since the offer does not directly accept E-Mails?

This might reveal itself as a design oversight.